src/clients/api/queries/getIsolatedPools/index.ts

```diff
@@ -15,7 +15,7 @@ import { GetIsolatedPoolsInput, GetIsolatedPoolsOutput } from './types';
 
 export type { GetIsolatedPoolsInput, GetIsolatedPoolsOutput } from './types';
 
-const POOL_REGISTRY_ADDRESS = getContractAddress('PoolRegistry');
+const POOL_REGISTRY_ADDRESS = getContractAddress('PoolRegistry_Proxy');
```

Comment:
I'd keep PoolRegistry, both should return the same address

Reply:
They should yeah, but my understanding is that using the proxy should be safer since if a new Pool Registry contract is deployed then the proxy will automatically point to it, meaning we won't have to redeploy the frontend.
Perhaps I'm wrong?

Reply:
AFAIK PoolRegistry_Proxy is autogenerated and have the same address as PoolRegistry, meaning that whenever we update the package, the address returned by both of them will be the same. I think it's worth confirming that with contracts to have clarity.